draft-ietf-quic-tls.md

```diff
@@ -532,9 +532,10 @@ older than 1.3 is negotiated.
 
 ## ClientHello Size
 
-QUIC requires that the initial handshake packet from a client fit within a
-single packet of at least 1280 octets.  With framing and packet overheads this
-value could be reduced.
+QUIC requires that the initial handshake packet from a client fit within the
+payload of a single packet.  A minimum-sized initial packet in IPv6 has space
+for a single TLS record of 1197 octets once the QUIC header and minimal framing
+is added.
```

Comment:
A STREAM frame header can be two octets.  And then there is the FNV-1a hash, which I know isn't in the current draft, but I'm trying to avoid having to update this once that lands.

I really only wanted to say point 2 here.  I'll take another stab at it.